system-test/pubsub.ts

```diff
@@ -134,7 +193,10 @@ describe('pubsub', () => {
     it('should honor the autoCreate option', done => {
       const topic = pubsub.topic(generateTopicName());
 
-      topic.get({autoCreate: true}, done);
+      topic.get({autoCreate: true}, (err, _topic, _apiResponse) => {
```

Comment:
Why did we need to change this?

Reply:
In my view need to check for any error.

Reply:
If an error gets passed into `done` the test will fail, so the test is essentially the same - this one is just a little more verbose. I think including `topic` and `apiResponse` as params should/will trigger a lint error since they are unused.

Reply:
Got it. Thanks for info.